packages/recommend-react/src/index.ts

```diff
@@ -7,3 +7,5 @@ export * from './useRecommendations';
 export * from './useRelatedProducts';
 export * from './useTrendingFacets';
 export * from './useTrendingItems';
+export { Recommend } from './Recommend';
```

Comment:
what's the reason of not using `export *`? If there are exported internals, those could be moved to a different file.

Reply:
This was discussed with @francoischalifour as we don't want to export internal details of how the context works to the end users, while keeping them accessible internally.

I don't think file structure should change here.